packages/error-reporting/package.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Why would we need to ignore the errors directory for unit-tests -- couldn't we run the errors tests as well?